src/main.py

```diff
@@ -1,13 +1,18 @@
-import uvicorn
-
-from src.routes.api import app  # noqa: F401 - uvicorn needs to import the app
-
-if __name__ == "__main__":
-    config = uvicorn.Config(
-        app="main:app",
-        port=8000,
-        log_level="info",
-        use_colors=True,
-    )
-    server = uvicorn.Server(config)
-    server.run()
+from fastapi import FastAPI
+from starlette.responses import StreamingResponse
+
+from services.chat import init_chat, process_chat_request
+
+app = FastAPI()
+
+
+@app.get("/chat/init")
+async def init() -> dict:
+    """ Endpoint to initialize the chat with the Kyma companion """
+    return await init_chat()
+
+
+@app.get("/chat")
+async def chat() -> StreamingResponse:
+    """ Endpoint to chat with the Kyma companion """
+    return StreamingResponse(process_chat_request(), media_type='text/event-stream')
```

Comment:
Why don't we keep the routes separately using the `APIRouter` class? and in main we can include the route with `/api/v1` prefix, so we can easily later version the APIs and also introduce new set of APIs.